modules/express/src/clientRoutes.ts

```diff
@@ -729,7 +729,11 @@ async function handleV2SendMany(req: express.Request) {
   req.body.reqId = reqId;
   let result;
   try {
-    result = await wallet.sendMany(createSendParams(req));
+    if (wallet._wallet.multisigType === 'tss') {
+      result = await wallet.sendMany(createTSSSendParams(req));
```

Comment:
can you verify this tss param will work for ECDSA?

Reply:
we do not support ECDSA yet.